### ISSUE TITLE
Metatags

### ISSUE THREAD

**Issue by MarauderXtreme** [fsr/ese-webseite#24]
We should do some SEO for this site. I've searched a few keywords for this site and I'm pretty impressed that the ranking is quite good. But we can do better.
I've talked about this with @JustusAdam and he suggested that we could use these "frontmatter"(?) thingys to override default settings for every sub page if needed. Maybe we should evaluate this.

What we should do now:
1. Add somehow some basic metatags (abstract, keywords and some other OG tags)
2. Create some files (e.g. apple-touch-icons for our friends with special needs)
3. Find a way to overwrite these defaults for subsites if needed (descriptions, keyword or whatever)
4. i18n

**Comment by MarauderXtreme** [comment]
@JustusAdam would you kindly look ove my code? I'm desperate. I have no idea what is wrong and why this will not work. At least I think I have writte the code correct but somehow it looks like the values are not there.

**Comment by JustusAdam** [comment]
I fixed your code, and I changed the loops a bit, made it more automated.

**Comment by MarauderXtreme** [comment]
Thank you. This looks good.
First I wanted it as simple as possible but then it growed and I think that is the point where I screwed up the yaml.

**Comment by JustusAdam** [comment]
You may want to look at the two commits separately. 66c22501f840a2101dad7a0701c9c1355a82e279 Fixes the issue you were having with the YAML data, in b7b1388e9b552bd1badfcae1346fdf7a833de132 I merged the loops to avoid code repetition.

**Comment by MarauderXtreme** [comment]
Yep. Already noticed. My problem was afaics the "-" after every type. No idea why I included it.

**Comment by JustusAdam** [comment]
Exactly. that made it a list, rather than an object, which is what you wanted.

**Comment by MarauderXtreme** [comment]
Maybe we should now make a list with metatags that are a good idea for this site.
* description: already exists
* abstract
* keywords
* og:title
* og:description

* og:locale
** og:locale:alternate


* og:image
** og:image:secure_url
** og:image:type

* og:site_name
* og:url
* og:type
* HandheldFriendly: set to true

Maybe someone can thnk about the correct twitter card tags and some of these Google Tags.

**Comment by MarauderXtreme** [comment]
I implemented the said tags but unfortunately I am not able to handle yaml files. @JustusAdam help. I am lost. Would you kindly look over 28f12c0d61ef9f5700340944b9c84184ff4ccd18 and a7affa40630150139e4e019125fc161a7fc33d4b.
Where did I go wrong?

**Comment by JustusAdam** [comment]
You weren't listening when I explained how this is done. You add overriding values to the front matter, and set the defaults like [this](https://jekyllrb.com/docs/configuration/#front-matter-defaults). Don't nest them, aka `og.locale`, use flat values `og-locale` otherwise the defaults will be a pain.

If you set it up this way you can save all the if-else blocks and just write `{{ page.og-locale }}`.

And why did you indent the entire file? How am I supposed to read the diff if you make such an unnecessary change? Furthermore it doesn't fit with the rest of the code now in terms of indentation.

**Comment by MarauderXtreme** [comment]
You did not realy explain this anytime. This example is near to useless for me since it only describes already implemented things. I don't see how I should bring this to custom keys because in the example are mostly predefined things. What scope do I have to use?

The nested OpenGraph Tags were intented to be used with key and value printed to the template. But it looks like it will not work. Flat values will be very unhandy since they can get very long. OG Tags have a kind of structure with their : syntax so it would be more intuitive to nest them. But if you say so.

The indentation was made by my sublime text. I do not think that this is not conform with everything else since I almost use tabs to indent and this shows up in all files I have written.

**Comment by JustusAdam** [comment]
Have you even tried it?

**Comment by MarauderXtreme** [comment]
Yep but I have stoped because I get nothing but parse errors.

**Comment by JustusAdam** [comment]
Then fix your praise errors or start explaining to me what the real problem is, rather than just writing random code.

**Comment by MarauderXtreme** [comment]
I already explained the real problem. i have no idea how to do it.
I told you to look over the code and tell me where I did go wrong. But instead you point to a close to useless doc. It completely lacks of an explanation how I can access values defined under these scope thingys.

I now have figured out what was causing the trouble and I fixed it. The real problem was the multiline ">" markup. Well and that you have showed me and it did not work. ^^

Now feel free to show me how it should be done in jekyll.
